exporter/newrelicexporter/transformer.go

```diff
@@ -63,7 +63,7 @@ func newTransformer(logger *zap.Logger, buildInfo *component.BuildInfo, details
 	return &transformer{logger: logger, OverrideAttributes: overrideAttributes, details: details}
 }
 
-func (t *transformer) CommonAttributes(resource pdata.Resource, lib pdata.InstrumentationLibrary) map[string]interface{} {
+func (t *transformer) CommonAttributes(resource pdata.Resource, lib pdata.InstrumentationScope) map[string]interface{} {
```

Comment:
This changes the signature of an exported method (though on an unexported type).  Do we know whether this is supposed to be implementing an interface?

Reply:
This is a good question, maybe @alanwest @jack-berg @nrcventura can help here.

Reply:
This change looks safe.

Reply:
I agree that it looks safe. That exported method is not implementing an interface. The transformer type is only meant to be used internally.